server/src/main/java/org/apache/iotdb/db/engine/storagegroup/TsFileProcessorInfo.java

```diff
@@ -31,7 +29,7 @@
 
   public TsFileProcessorInfo(StorageGroupInfo storageGroupInfo) {
     this.storageGroupInfo = storageGroupInfo;
-    this.memCost = IoTDBDescriptor.getInstance().getConfig().getWalBufferSize();
+    this.memCost = 0L;
```

Comment:
Why do not count the wal buffer size?

Reply:
Since 0.12, wal buffer uses off-heap memory. We shouldn't count it in writing memory which is a part of JVM on-heap memory....